### PR TITLE
[CP] Release 4.7.0.rc2

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@sjha4 can I get a review?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,130 @@
...
+ * Importing a custom repository with different label but same name causes validation error ([#35425](https://projects.theforeman.org/issues/35425), [cf159cad](https://github.com/Katello/katello.git/commit/cf159cadb0b306faea429aba0af5fa07ad45b105))
+ * Pathing issue on exports ([#35410](https://projects.theforeman.org/issues/35410), [491e8af0](https://github.com/Katello/katello.git/commit/491e8af044cae8da1cb7b2c77f33516dda79aab3))
+ * Need to be able to provide custom cert for ISS for Red Hat CDN ([#35296](https://projects.theforeman.org/issues/35296), [a0e63cb0](https://github.com/Katello/katello.git/commit/a0e63cb063a07646ef079acfab762a6e7435b110), [6a47ac35](https://github.com/Katello/hammer-cli-katello.git/commit/6a47ac3546be9aafa2533e09b9ca4b51fd449f43))
+
+### Container
+ * Getting "undefined method `schema_version' for nil:NilClass" while syncing from quay.io ([#35709](https://projects.theforeman.org/issues/35709), [475388ea](https://github.com/Katello/katello.git/commit/475388eaef0796339326b17b8362b5015c719f9c))
```
Comment (jeremylenz) [inline CHANGELOG.md:83]: Ruby error messages break Markdown.. :confounded:

Comment (jeremylenz) [inline CHANGELOG.md:83]: Ruby error messages break Markdown.. :confounded:

Comment (chris1984) [reply CHANGELOG.md:83]: It looks fine in Chrome, are you seeing it not render at all?

Comment (chris1984) [reply CHANGELOG.md:83]: I will push up a rebased commit, I removed the `'` `` and `nil:`

**Comment by chris1984** [comment]
@jeremylenz tests are green, can i get another review please

**Review by jeremylenz — APPROVED** [review]

**Pull request merged**